packages/builder/src/internal/core/project-structure.ts

```diff
@@ -1,22 +1,14 @@
 import findUp from "find-up";
 
-import { isTypescriptSupported } from "./typescript-support";
 const JS_CONFIG_FILENAME = "builder.config.js";
 const TS_CONFIG_FILENAME = "builder.config.ts";
 
 export function isCwdInsideProject() : boolean {
-  return (
-    Boolean(findUp.sync(JS_CONFIG_FILENAME)) ||
-      isTypescriptSupported() && Boolean(findUp.sync(TS_CONFIG_FILENAME))
-  );
+  return Boolean(findUp.sync(JS_CONFIG_FILENAME));
 }
 
 export function getUserConfigPath(): string | undefined {
-  const tsConfigPath = findUp.sync(TS_CONFIG_FILENAME);
-  if (tsConfigPath) {
-    if (isTypescriptSupported()) {
-      return tsConfigPath;
-    }
+  if (findUp.sync(TS_CONFIG_FILENAME)) {
```

Comment:
You don't only disable TS support but crash if the user wants to actually use TS using `ts-node` as executor.

And you'll also crash if the user will want to have an unrelated file `algob.config.ts` that he won't want to load.

I would prefer to: 
1. check if `ts-node` is loaded.
2. look for JS config if it's not loaded. Fail if no config is found.
3. look for TS config if it's loaded.

- [ ]  Fix the crash that will occur when using TS files.

Reply:
We will see if we need a `ts-node` support. For the moment there is no benefit of it (and as already noted it significantly increases the runtime, making the too usage a bit clunky) so we will DROP it. We can simply enable it in the future if needed and users will be able to compare 3 versions.

Reply:
I removed the TS config check with the `throw` statement.